src/components/library/LibraryControlBar.tsx

```diff
@@ -96,6 +104,31 @@ const LibraryControlBar: React.FC<Props> = (props: Props) => {
     return prefix;
   };
 
+  /**
+   * Get a displayable string for the current libraryType value.
+   * @returns a user-friendly representation of the libraryType prop
+   */
+  const getLibraryTypeText = () => {
```

Comment:
I think this prefix & variable/enum name (including in the setting) should be changed to "View" / "LibraryView".

Reply:
Fix with latest commit